src/runtime/static.ts

```diff
@@ -92,11 +92,10 @@ export default eventHandler((event) => {
     event.res.setHeader("Content-Length", asset.size);
   }
 
-  // TODO: Asset dir cache control
-  // if (isBuildAsset) {
-  // const TWO_DAYS = 2 * 60 * 60 * 24
-  // event.res.setHeader('Cache-Control', `max-age=${TWO_DAYS}, immutable`)
-  // }
+  if (isPublicAssetURL(id) && !event.res.getHeader("Cache-Control")) {
```

Comment:
for a public asset base of `/` (which would be the case with Nuxt's `/public` directory) this will cache all html/assets, including things like `robots.txt`, `favico.ico` or even `/about/index.html`. I think that is probably too broad a net to cast.

Reply:
I think @pi0 has some thoughts about this, but maybe allow cache headers to be set per asset folder? So Nuxt could push in `/_nuxt`, for example, with the appropriate long cache headers.